JourneysSquad/NavigationBarForHistoryOfOperationView.swift

```diff
@@ -0,0 +1,32 @@
+import SwiftUI
+
+/// View with a navigation bar, a title and a toolbar item.
+struct NavigationBarForHistoryOfOperationView: View {
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Text("Any other view can be placed here")
+            }
+            .navigationTitle("History of operations")
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                Text("Add")
+                    .font(.system(size: fontBodySize, design: .default))
```

Comment:
```suggestion
                    .font(.system(size: fontBodySize)
```
nit: 
Can we write in such a way?